### PR TITLE
BI-11398 navigate delivery options

### PR DESCRIPTION
**Pull request by aokunade**
BI-11398 navigate delivery options

### CODE REVIEW

File: src/controllers/certificates/delivery.options.controller.ts [hunk 1]
```diff
@@ -33,9 +28,13 @@ export const render = async (req: Request, res: Response, next: NextFunction): P
         const userId = getUserId(req.session);
         const accessToken: string = getAccessToken(req.session);
         const certificateItem: CertificateItem = await getCertificateItem(accessToken, req.params.certificateId);
+        const expressCost =  "50";
```
Comment (ethatcher83) [inline src/controllers/certificates/delivery.options.controller.ts:31]: Doesn't look like you are covering the output of these values for the error page

File: src/controllers/certified-copies/delivery.options.controller.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        const accessToken: string = getAccessToken(req.session);
+        const certifiedCopyItem: CertifiedCopyItem = await getCertifiedCopyItem(accessToken, req.params.certifiedCopyId);
+        const companyNumber: string = certifiedCopyItem.companyNumber;
+        const filingType: string = certifiedCopyItem.itemOptions.filingHistoryDocuments[0].filingHistoryType;
+        const expressCost = filingType === "NEWINC" ? "100" : "50";
+        const standardCost = filingType === "NEWINC" ? "50" : "15";
+        logger.info(`Get certified copy item, id=${certifiedCopyItem.id}, user_id=${userId}, company_number=${certifiedCopyItem.companyNumber}`);
+        return res.render(DELIVERY_OPTIONS, {
+            DISPATCH_DAYS,
+            expressCost,
```
Comment (ethatcher83) [inline src/controllers/certified-copies/delivery.options.controller.ts:36]: Doesn't look like you are covering the output of these values for the error page. I'd also suggest making these values all caps like DISPATCH_DAYS

File: src/client/api.client.ts [hunk 3]
```diff
@@ -119,6 +119,18 @@ export const getCertifiedCopyItem = async (oAuth: string, certifiedCopyId: strin
     return certifiedCopyItemResource.resource as CertifiedCopyItem;
 };
 
+export const patchCertifiedCopyItem = async (
```
Comment (ethatcher83) [inline src/client/api.client.ts:122]: Tests will need to be added to test this patch functionality

File: src/controllers/certified-copies/delivery.options.controller.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
+import { NextFunction, Request, Response } from "express";
```
Comment (ethatcher83) [inline src/controllers/certified-copies/delivery.options.controller.ts:1]: This controller will require test coverage

**Review by hwilliams1 — CHANGES_REQUESTED** [review]
Doesn't look like there are any unit tests on this. Can unit tests be added please?

**Review by hwilliams1 — APPROVED** [review]

**Pull request merged**